lib/shopify_api/context.rb

```diff
@@ -150,6 +153,11 @@ def activate_session(session)
       def deactivate_session
         @active_session.value = nil
       end
+
+      sig { returns(String) }
+      def host_scheme
+        T.must(URI.parse(T.must(host)).scheme)
```

Comment:
Would it make sense to store `host` as a `URI` to make it easier to handle later?